test/suites/storage/pvc-resize.robot

```diff
@@ -0,0 +1,35 @@
+*** Settings ***
+Documentation       Tests related to a functional PVC after reboot
+
+Resource            ../../resources/common.resource
+Resource            ../../resources/oc.resource
+Resource            ../../resources/ostree-health.resource
+
+Suite Setup         Setup Suite With Namespace
```

Comment:
Should we have a corresponding `Suite Teardown`? i.e. `Teardown Suite With Namespace`